views/index.erb

```diff
@@ -0,0 +1,70 @@
+<html>
+<head>
+  <meta charset="utf-8">
+  <meta name="ROBOTS" content="NOODP">
+  <meta name="viewport" content="width=920">
+  <title>Rouge</title>
+  <link rel="stylesheet" type="text/css" media="all" href="https://necolas.github.io/normalize.css/latest/normalize.css">
+  <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Raleway:800,500|Source+Code+Pro:400,700">
+  <link rel="stylesheet" type="text/css" media="all" href="/assets/application.css">
+  <link rel="author" href="humans.txt">
+  <link rel="shortcut icon" type="image/x-icon" href="/assets/favicon.ico">
+</head>
+<body class="home_pages home_pages-show">
+  <header>
+    <h1 class="main">
+      <a href="/">{ Rouge }</a>
+    </h1>
+    <h2>An elegant, extendable code highlighter written in pure Ruby.</h2>
+    <h4>Available on <a target="_blank" href="https://github.com/rouge-ruby/rouge">GitHub</a></h4>
```

Comment:
`target="_blank"` was in the original source, but if I did it over again, I'd remove it. I find that I don't want to remove the user's choice on when to open a new tab or not. Also, the target attribute has [apparently been deprecated](https://css-tricks.com/list-of-depreciated-elements-still-in-widespread-use/). Although I doubt it will stop working anytime soon.

There are lots of arguments [for](https://uxmovement.com/navigation/why-external-links-should-open-in-new-tabs/) and [against](https://marco.org/2014/01/10/target-blank) so I'll leave it up to you, but I wanted to through my two cents in.

Reply:
You're right, I think this should be removed as well. I'll do that and push another commit.